### ISSUE TITLE
Center Covers for other aspect ratios

### ISSUE THREAD

**Issue by davidkgriggs** [davidkgriggs/PS5ish#2]

**Comment by ToMattBan** [comment]
When I add a game less famous (like these one: https://www.bigfishgames.com/games/14672/viking-heroes-collectors-edition/?pc) Is very difficult to found a good cover, so I choose a good image to use as a cover. With the default Playnite theme, I have a choice to "Fill & Center" (Something like these, I don't use Playnite in English), this option, like the name, fill the entire cover and center the image. But when I installed the PS5ish theme, all these "imperfect" covers are aligned in the left. I'm using a 3:4 aspect ratio.

**Comment by davidkgriggs** [comment]
> When I add a game less famous (like these one: https://www.bigfishgames.com/games/14672/viking-heroes-collectors-edition/?pc) Is very difficult to found a good cover, so I choose a good image to use as a cover. With the default Playnite theme, I have a choice to "Fill & Center" (Something like these, I don't use Playnite in English), this option, like the name, fill the entire cover and center the image. But when I installed the PS5ish theme, all these "imperfect" covers are aligned in the left. I'm using a 3:4 aspect ratio.

In Desktop Settings in Grid View is your Stretch Mode set to Fill? Try setting it to Fill if it isn't and let me know because that sorts it for me.

**Comment by ToMattBan** [comment]
I've changed the language to English here to help you to help me hahah. The Fill works, but the cover become pressed and weird.  The opyion I use is "Uniform to Fill". Here you can see what I am saying. Usong the crunchyroll you note the cover are not sligned on center like in the normal mode.

![IMG20210127230105](https://user-images.githubusercontent.com/50156748/106079244-9c44b500-60f3-11eb-8eb1-51bc5b2de70e.jpg)
![IMG20210127230127](https://user-images.githubusercontent.com/50156748/106079248-9e0e7880-60f3-11eb-8878-652d17e4987c.jpg)

**Comment by davidkgriggs** [comment]
Do you always use 3:4 (IGDB) setting for grid view? You could Crop the Crunchy Roll image to be a 3:4 aspect ratio and then with Fill it will work fine. I think because I'm setting strict Width and Height so that it can have rounded corners it won't work correctly with images that are not the right aspect ratio to what you have set in the Grid View

**Comment by ToMattBan** [comment]
Yeah, I think 3:4 have a better look, so I always use it. But if I use a 1:1 the result will be the same, because some games less famous don't have much images to use as cover. Crop the images really will fix my problem, but when I have more than a few games this can be a mess. Can you tell me where you change the corners, so maybe I can fix it?

**Comment by davidkgriggs** [comment]
Can you send me the image you're using so I can have a play with it?

**Comment by ToMattBan** [comment]
I can't refind the image, I use the Playnite function to search on Google. But this one is very similar:

https://i2.wp.com/www.drunkendwarf.net/wp-content/uploads/2016/07/crunchyroll_wallpaper.jpg

**Comment by davidkgriggs** [comment]
After a lot of investigation I really don't think this is possible with my set-up, but there is a way around it, there is an extension to resize images, it can also crop them and do all sorts: https://playnite.link/forum/thread-565.html
If you use this you should be able to get all your images to 3:4 aspect ratio and they will then be compatible with all themes.
